ts/components/conversation/composition/EmojiQuickResult.tsx

```diff
@@ -28,16 +28,18 @@ export const searchEmojiForQuery = (query: string): Array<SuggestionDataItem> =>
   if (query.length === 0 || !emojiIndex) {
     return [];
   }
-  const results = emojiIndex.search(query);
+  const results1 = emojiIndex.search(":"+query) || [];
```

Comment:
matching emoticons requires searching with colon - e.g. typing ':)'  would trigger the selector panel, and search ')', whereas the emoticon's code is ':)'